pkg/operator/configobservation/images/observe_images.go

```diff
@@ -1,42 +1,163 @@
 package images
 
 import (
+	"bytes"
+	"encoding/json"
+
 	"github.com/golang/glog"
 
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 
-	"github.com/openshift/cluster-openshift-apiserver-operator/pkg/operator/configobservation"
 	"github.com/openshift/library-go/pkg/operator/configobserver"
+
```

Comment:
nit: unusual newline

Reply:
I separate imports from the same repo from imports from other repos.  Call it a personal quirk but i find it makes it easier to scan what's being imported from where.

I basically group imports into:

```
system

random thirdparties

k8s

openshift(other repos)

current repo
```